packages/components/src/card/card-body/README.md

```diff
@@ -1,6 +1,6 @@
 # CardBody
 
-CardBody renders an optional content area for a [`<Card />`](../).
+`CardBody` renders an optional content area for a [`Card`](../card/README.md). Multiple `CardBody` components can be used within `Card` if needed.
```

Comment:
Apparently relative paths don't work for links to other markdown files, at least according to @gziolo: https://github.com/WordPress/gutenberg/pull/31297#discussion_r623120634

Reply:
I was not aware of this issue! Updated all paths in https://github.com/WordPress/gutenberg/pull/32566/commits/94dd62dafebab464b535cdea3c6182e1ca4d5629